### DISCUSSION TITLE
unable to connect to crazyradio on M1 Mac

### DISCUSSION THREAD

**Discussion by sawyerbfuller** [bitcraze/discussions#470] [General]
Sincerest apologies for the noise if this is just a hardware issue. I can't seem to get `cfclient` to find my crazyradio. When I click "select an interface" at the top left, the radio doesn't appear like [in the getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/): 
<img width="659" alt="image" src="https://user-images.githubusercontent.com/58706249/207510228-6107e134-5881-4c69-9476-dc9bba3b8526.png">

Here is a terminal dump: 
<img width="569" alt="image" src="https://user-images.githubusercontent.com/58706249/207510502-375c6db2-066c-4763-8fb0-88eca3e07c9c.png">

and later, 
<img width="554" alt="image" src="https://user-images.githubusercontent.com/58706249/207510969-06888d34-4fe4-4419-8e92-0327e91087f2.png">

and my system Information seems to think it's connected (and I get a red+green LED flash when I do):
<img width="435" alt="image" src="https://user-images.githubusercontent.com/58706249/207510616-39292eda-7db7-403c-93fe-924a8839c4ca.png">

This looks a lot like bitcraze/crazyflie-clients-python#291 - do I need to re-flash the radio? (it's been sitting in a box for a couple of years - thanks Covid)

Hardware: Crazyradio PA, M1 Macbook Pro

Thanks

**Comment by ataffanel** [comment]
Hi,

Nothing looks off in your logs and system info. The radio has already been updated (most likely to run with ROS/Crazyswarm) but it should work fine.

Are you clicking the "Scan" button, and when doing so does the LED light up, the RED LED should light up on the radio and the green one should blink fast if a Crazyflie is detected. The interface drop-down menu should then show one line per detected Crazyflie.

**Comment by sawyerbfuller** [comment]
OK, figured it out. Indeed the light turned red when I clicked "scan". 

That was enough to know that it wasn't the crazyradio, and then I realized the crazyflie I have was programmed to have a different address. 

I think I was expecting the "interface" box to indicate which crazyradio I was using -- or that I was using one at all -- and the "address" box below to correspond to the crazyflie's address. But somehow it seems that you can select the correct interface only once you have the address correct below. Anyway, settled, thanks!
